docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -205,125 +181,141 @@ You should see the contents of the project's `data/` directory printed to your t
-- In the `Entry Point` field, enter `databricks_run`. This is the name of the [entry point](#create-an-entry-point-for-databricks) to run your package from.
-- Ensure the job cluster you created in step two is selected in the dropdown menu for the `Cluster` field.
-- In the `Dependent libraries` field, click `Add` and upload [your project's `.whl` file](#package-your-project), making sure that the radio buttons for `Upload` and `Python Whl` are selected for the `Library Source` and `Library Type` fields.
-- In the `Parameters` field, enter the following list of runtime options:
+### Create asset bundles files
+
+
+
+#### 1. Create the bundle
+
+Create a folder called `assets` in the root directory containing the file `assets/batch-inference-workflow-asset.yml`, this name is just an example:
+```yaml
+common_permissions: &permissions
+  permissions:
+    - level: CAN_VIEW
+      group_name: users
+
+experimental:
+  python_wheel_wrapper: true
+
+artifacts:
+  default:
+    type: whl
+    build: kedro package
+    path: ..
+resources:
+  jobs:
+    iris-databricks-job: #used to run job from CLI
+      name: "iris-databricks" #name of the job in databricks
+      tasks:
+        - task_key: kedro_run
+          existing_cluster_id: ${var.existing_cluster_id}
+          python_wheel_task:
+            package_name: iris #<package_name>
+            entry_point: databricks_run
+            named_parameters:
+              --conf-source: /Workspace${workspace.file_path}/conf
+              --package-name: iris #<package_name>
+              --env: ${var.environment}
+          libraries:
+            - whl: ../dist/*.whl
+      schedule:
+        quartz_cron_expression: "0 0 11 * * ?" # daily at 11am
+        timezone_id: UTC
+      <<: *permissions
+```
+This file provides a template for a job utilizing Kedro as a package, and it allows for multiple definitions per job. For instance, you could have one definition per pipeline.
+
+
+In this example, an existing cluster ID is being utilized, which can facilitate fast iteration. However, for a productive deployment, a job cluster is required. To enable ephemeral clusters, the YAML file (`assets/batch-inference-workflow-asset.yml`) needs to be modified to use a `job_cluster_key` instead of an `existing_cluster_id`.
+
+```yaml
+jobs:
+    iris-databricks-job: #used to run job from CLI
+      name: "iris-databricks"  #name of the job in databricks
+      job_clusters:
+        - job_cluster_key: job_cluster
+          new_cluster:
+              #Azure nodes
+              node_type_id: Standard_DS3_v2
+              spark_version: 14.3.x-scala2.12
+      tasks:
+        - task_key: kedro_run
+          job_cluster_key: job_cluster
+          python_wheel_task:
+            package_name: iris #<package_name>
+            entry_point: databricks_run
+            named_parameters:
+              --conf-source: /Workspace${workspace.file_path}/conf
```

Comment:
What about passing the `project-path` instead and the change the working directory in the `databricks_run.py` script? I do not know if this a good option but sometimes I have more files that I need in the project path, like sql files that are not reachable in databricks unless I change the working directory or I specify the full path.